common/changes/office-ui-fabric-react/edwl-2018-09-removeClassNamesFunctionResizeGroup_2018-09-17-19-37.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "ResizeGroup: Remove classNamesFunction from ResizeGroup",
```

Comment:
The change is a little more involved than that. You also deprecated (and render non-functional) the styles prop of ResizeGroup. This is a very notable change that should be called out.

Reply:
Ok, will update soon.  Is this still considered a "patch" change or "minor"?